### PR TITLE
fix: Don't show pin button in input panel when there's binary data

### PR DESCRIPTION
**Pull request by mutdmour**
## Summary

Don't show pin button in input panel when there's binary data
image

## Review / Merge checklist

- [x] PR title and summary are descriptive. ([conventions](../blob/master/.github/pull_request_title_conventions.md)) 
- [ ] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.
- [x] Tests included. 
- [ ] PR Labeled with `release/backport` (if the PR is an urgent fix that needs to be backported)

### CODE REVIEW

File: packages/editor-ui/src/components/RunData.vue [hunk 1]
```diff
@@ -269,6 +269,13 @@ export default defineComponent({
 				!(this.binaryData && this.binaryData.length > 0)
 			);
 		},
+		showPinDataButton(): boolean {
```
Comment (netroy) [inline packages/editor-ui/src/components/RunData.vue:272]: This could be written to be more readable. WDYT?

Comment (mutdmour) [reply packages/editor-ui/src/components/RunData.vue:272]: Sure ofc yes

**Review by netroy — DISMISSED** [review]

**Review by MiloradFilipovic — APPROVED** [review]
Looks good to me.
Nit: PR title is a bit confusing since the button is shown, just disabled so we may want to get that in order for the changelog, otherwise good to go

**Comment by janober** [comment]
Got released with [`n8n@1.65.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.65.0)

**Pull request merged**